content/docs/setup/kubernetes/upgrading-istio/index.md

```diff
@@ -255,3 +255,61 @@ spec:
     tls:
       mode: DISABLE
 {{< /text >}}
+
+## Migrating the `RbacConfig` to `ClusterRbacConfig`
+
+The `RbacConfig` has a [bug](https://github.com/istio/istio/issues/8825). The bug reduces the scope of
+the object to be namespace-scoped in some cases. The `ClusterRbacConfig` follows the exact same
+specification as the `RbacConfig` but with correct cluster scope implementation.
```

Comment:
This text explains the bug and the fact that `ClusterRbacConfig` does not have the buggy behavior. That implies why the migration is needed but fails to be explicit or to indicate: When should readers migrate? Under what conditions does the bug impact them?

Reply:
everyone should migrate if they are currently using RbacConfig

Reply:
I do not see that statement anywhere in the doc. Please add a clear statement at the beginning of the section.